contrib/platform/src/com/sun/jna/platform/win32/VersionUtil.java

```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2015 Michael Freeman, All Rights Reserved
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.  
+ */
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.Memory;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.platform.win32.VerRsrc.VS_FIXEDFILEINFO;
+import com.sun.jna.platform.win32.Version;
+import com.sun.jna.platform.win32.Win32Exception;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.PointerByReference;
+
+/**
+ * Reads Windows Version info from files (the version details you can see by
+ * right-clicking and choosing properties)
+ * 
+ * @author mlfreeman[at]gmail.com
+ */
+public class VersionUtil {
+
+    /**
+     * Gets the file's version number info
+     * 
+     * @param filePath
+     *            The path to the file
+     * @return Either an array of {major, minor, build, revision} or null if no
+     *         version info could be obtained.
+     */
+    public static int[] getFileVersionNumbers(String filePath) {
+        IntByReference dwDummy = new IntByReference();
```

Comment:
How does this sound as an option?
1. Return the `VS_FIXEDFILEINFO` that I already have in the function.
2. Add the following four getters to it: 
```java
        public int getFileVersionMajor() {
            return dwFileVersionMS.intValue() >>> 16;
        }
        
        public int getFileVersionMinor() {
            return dwFileVersionMS.intValue() & 0xffff;
        }
        
        public int getFileVersionRevision() {
            return dwFileVersionLS.intValue() >>> 16;
        }
        
        public int getFileVersionBuild() {
            return dwFileVersionLS.intValue() & 0xffff;
        }
```
3. Change the method name to `getFileVersionInfo()`